pkg/kubelet/apis/deviceplugin/v1beta1/api.pb.go

```diff
@@ -152,10 +153,27 @@ func (m *ListAndWatchResponse) GetDevices() []*Device {
 	return nil
 }
 
+type TopologyInfo struct {
+	Socket int64 `protobuf:"varint,1,opt,name=socket,proto3" json:"socket,omitempty"`
+}
+
+func (m *TopologyInfo) Reset()                    { *m = TopologyInfo{} }
+func (*TopologyInfo) ProtoMessage()               {}
+func (*TopologyInfo) Descriptor() ([]byte, []int) { return fileDescriptorApi, []int{4} }
+
+func (m *TopologyInfo) GetSocket() int64 {
+	if m != nil {
+		return m.Socket
+	}
+	return 0
```

Comment:
Will this code lead to situation where socket will be always 0 to the devices which are socket agnostic (or device plugin does not set socket to any value) ?

Reply:
@kad if plugin does not set the value then it will be defaulted to nil. Just tested it